ui/package.json

```diff
@@ -117,7 +121,7 @@
     "serialize-javascript": "^3.1.0",
     "stackdriver-errors-js": "^0.4.0",
     "stacktrace-js": "^2.0.0",
-    "ts-jest": "^25.3.0",
+    "ts-jest": "^26.5.4",
```

Comment:
Should this be moved to devDependencies?

Reply:
Good catch. Done.